src/app/dataset-view/additional-components/metadata-component/components/form-components/event-time-field/event-time-field.component.html

```diff
@@ -0,0 +1,57 @@
+<div [formGroup]="form">
+    <ng-container formGroupName="eventTime">
+        <div class="d-flex">
+            <app-tooltip-icon [tooltip]="tooltip"></app-tooltip-icon>
+            <label class="form-label">{{ label }} </label><br />
+        </div>
+        <div class="ms-4">
+            <div class="form-check">
+                <input
+                    class="form-check-input"
+                    type="radio"
+                    formControlName="kind"
+                    [name]="kindNameControl"
+                    type="radio"
+                    [value]="eventTimeSourceKind.FROM_METADATA"
+                />
+                <label class="form-check-label" for=""> From metadata </label>
+            </div>
+            <div class="form-check mt-1">
+                <input
+                    class="form-check-input"
+                    type="radio"
+                    [name]="kindNameControl"
+                    formControlName="kind"
+                    type="radio"
+                    [value]="eventTimeSourceKind.FROM_PATH"
+                />
+                <label class="form-check-label" for=""> From path </label>
+            </div>
```

Comment:
These choices should have a help hint too. The protocol defines some strings for this:

![image](https://user-images.githubusercontent.com/8875909/233832177-351f9116-c81d-497e-9f63-c55832ee0bd5.png)